impl/client/src/main/javascript/web/pentaho/ccc/visual/abstract.js

```diff
@@ -1411,16 +1411,18 @@ define([
         }
 
         // Set, before, whenever lineWidth is defined.
-        var dotSize = 16;
+        // var dotSize = 16;
```

Comment:
I thought only of providing some form of guidance to the developer. The _effective_ area (size) of the circle will still be `4 * 4 = 16`. However, one part of the radius will be given by the "circle" border and the other part by the "circle" fill.